tests/nav-spine_in-spine-no-list-style/OPS/package.opf

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Navigation Documents</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:creator>Dave Cramer</dc:creator>
+    <dc:date>2022-01-25</dc:date>
+    <dc:description>
+      The navigation document is also referenced in the spine; however, the TOC items in the Reading System's own control must not use the list style type of the navigation document.
+    </dc:description>
+    <dc:identifier id="pub-id">nav-spine_in-spine-no-list-style</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>TOC items ignore navigation list styling</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#confreq-nav-ol-style</meta>
+    <meta property="dcterms:modified">2022-01-25T00:00:00Z</meta>
+  </metadata>
+  <manifest>
+    <item id="content_001" href="content_001.xhtml" media-type="application/xhtml+xml" />
+    <item id="content_002" href="content_002.xhtml" media-type="application/xhtml+xml" />
+    <item id="nav" properties="nav" href="nav.xhtml" media-type="application/xhtml+xml" />
+  </manifest>
+  <spine>
+    <itemref idref="nav" />
```

Comment:
Is there a need for the nav to be in the spine? I think it confuses the test, because you'll see it once as content and once as UI, and the test only applies to the UI part (and the content part is likely to "fail" the test). Remove?

Reply:
I respectfully disagree on this one. 

In general, of course, there is no reason to put the nav into the spine (unless the nav file contains other things). But the very essence of the test is to show that one can add all kinds of styling to the navigation elements that are perfectly fine CSS statement that the Reading System can use when rendering the nav file as HTML but, on the other hand, the same styling is ignored when the navigation elements are displayed in the Reading System's chrome. To show that this navigation uses a unusual numbering of the (latin) text, ie, lower case greek characters.

I.e., the tests applies on the UI part _as opposed to the HTML styling_. Hence having the nav file in the spine.

Reply:
OK, that's fair, but in that case I think the text in nav.xhtml needs to be made clearer. It should basically say "doesn't matter what you see in the 'content area' [whatever we call that], although if the RS supports CSS it will probably have lower-greek numbering; look only at the navigation in the reading system UI."